### [CVE-2014-3730](https://cve.mitre.org/cgi-bin/cvename.cgi?name=CVE-2014-3730)
![](https://img.shields.io/static/v1?label=Product&message=n%2Fa&color=blue)
![](https://img.shields.io/static/v1?label=Version&message=n%2Fa&color=blue)
![](https://img.shields.io/static/v1?label=Vulnerability&message=n%2Fa&color=brighgreen)

### Description

The django.util.http.is_safe_url function in Django 1.4 before 1.4.13, 1.5 before 1.5.8, 1.6 before 1.6.5, and 1.7 before 1.7b4 does not properly validate URLs, which allows remote attackers to conduct open redirect attacks via a malformed URL, as demonstrated by "http:\\\djangoproject.com."

### POC

#### Reference
No PoCs from references.

#### Github
- https://github.com/ediskandarov/django-vulnerable
- https://github.com/emcpow2/django-vulnerable

